Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
```

Comment:
Is `breathecode/events/tests/urls/tests_me_event_checkin.py` being affected by these changes?

Reply:
I don't think so, I just ran the test again and I don´t see it any mistake

![image](https://github.com/user-attachments/assets/5c40a8b0-783c-4858-8790-324b5b01b917)